src/index.ts

```diff
@@ -112,7 +112,7 @@ const zodToTsNode = (
           literal = f.createNumericLiteral(literalValue)
           break
         case 'boolean':
-          if (literalValue === true) literal = f.createTrue()
+          if (literalValue) literal = f.createTrue()
```

Comment:
I prefer to be more explicit here - we actually are checking if it's true and not just truthy.

Reply:
one line above you've assured that the type is `boolean`.
Therefore, what else **truthy** value can it have besides `true`?